LayoutTests/fast/multicol/columns-shorthand-parsing-2-expected.txt

```diff
@@ -3,49 +3,56 @@ Test the behavior when 'auto' is part of the 'columns' property value. See http:
 On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
 
 
-PASS element.style.columnWidth is "auto"
+FAIL element.style.columnWidth should be auto. Was initial.
```

Comment:
What’s the deal with these? Is the test broken or the code? If it’s the test, can we fix the test?

Reply:
I think these failures are legit. We should not be setting the longhands to `initial`.

Reply:
I agree, will fix that.